index.html

```diff
@@ -9,19 +9,20 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <!-- Start: Header -->
     <div class="wrapper">
+      
+      <!-- Start: Header -->
       <header class="header">
         <a href="#">My Logo</a>
         <nav><i class="fa-solid fa-bars"></i></nav>
       </header>
      <!-- End: Header -->
 
-     <!-- Start: Main-->
-      <section>
+     <!-- Start: Main Section-->
+      <section class="main">
         <h1>I’m Hamza Baig. Glad to see <br> you!</h1>
         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
[OPTIONAL] Your styling is really good 👍. I only wanted to draw your attention to this little snippet right here in the inline comment. The color of the text in this paragraph seems to deviate a little from the figma design template 😅. It might be a good idea to set its color to match the color in the design template 😉.